action.yml

```
@@ -29,6 +29,9 @@ inputs:
   retry-exempt-status-codes:
     description: A comma separated list of status codes that will NOT be retried e.g. "400,500". No effect unless `retries` is set
     default: 400,401,403,404,422 # from https://github.com/octokit/plugin-retry.js/blob/9a2443746c350b3beedec35cf26e197ea318a261/src/index.ts#L14
+  base-url:
+    description: The root of the API URL of the GHES instance to which to connect.
```

What do you think about calling this the "GitHub REST API" URL to match the terminology we use to describe the `github.api_url` in our docs?

https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

---

Sure!

---

Maybe something like:
```suggestion
    description: An optional GitHub REST API to connect to a different GitHub instance.
```

---

And maybe include an example like `https://my.github-enterprise-server.com/api/v3` (is that the right form?)

---

Done!